package.json

```diff
@@ -16,29 +17,35 @@
   },
   "type": "module",
   "dependencies": {
-    "@sentry/node": "^6.11.0",
-    "@tinyhttp/proxy-addr": "^1.3.0",
-    "chrome-aws-lambda": "^10.1.0",
-    "cors": "^2.8.5",
+    "@sentry/node": "^6.12.0",
+    "@tinyhttp/proxy-addr": "^2.0.1",
     "dotenv": "^10.0.0",
     "flourite": "^1.1.0",
     "helmet": "^4.6.0",
-    "polka": "^0.5.2",
-    "puppeteer-core": "^10.2.0",
+    "polka": "^1.0.0-next.20",
     "rate-limiter-flexible": "^2.2.4",
-    "sharp": "^0.28.3",
-    "shiki": "^0.9.6",
-    "sirv": "^1.0.14"
+    "sharp": "^0.29.1",
+    "shiki": "^0.9.10",
```

Comment:
@elianiva kamu bump 2 package ini ya?

Reply:
![image](https://user-images.githubusercontent.com/51877647/133791232-99e67c15-a771-4a0c-a2b6-dbd7fade6235.png)

Reply:
hmm I didn't remember that. but ok.